package.json

```diff
@@ -56,6 +56,7 @@
     "jest": "25.2.4",
     "jest-watch-typeahead": "0.4.2",
     "node-fetch": "2.6.0",
+    "p-queue": "^6.4.0",
```

Comment:
This can be removed in latest alpha https://github.com/prisma/prisma/issues/2692

Reply:
Should be removable now that 2.0.0 has been released, right ?

Reply:
I'll revise this PR to be about 2.0 and yeah let's see.

Reply:
Ok works.